data/raw/keybindings.json

```diff
@@ -1475,6 +1475,13 @@
     "name": "Toggle auto start mod",
     "bindings": [ { "input_method": "keyboard", "key": "A" } ]
   },
+  {
+    "type": "keybinding",
+    "id": "SORT",
+    "category": "BIONICS",
+    "name": "Sort bionics list",
+    "bindings": [ { "input_method": "keyboard", "key": "S" } ]
```

Comment:
I would probably use `s` instead of `S` for sorting (to make behavior consistent across various menus) and use `S` for `TOGGLE_SAFE_FUEL` command.

Reply:
Good point; thanks